frontend/apps/web/src/pages/index.tsx

```diff
@@ -1,152 +1,188 @@
-                      name: kind,
-                      percentage,
-                    }))}
-                  />
+                  <Bar>
+                    <ProgressBar
+                      onClick={() => navigate(`system/${data.name}`)}
+                      data={data.testKindStats.percentages.map(
+                        ({ kind, percentage }) => ({ name: kind, percentage }),
+                      )}
+                    />
+                  </Bar>
                 )
               },
             },
             testStatus: {
               header: 'Test Status',
-              width: 222,
+              width: 325,
               Cell: ({ data }) => {
                 return (
-                  <ProgressBar
-                    onClick={() => navigate('/repository-details')}
-                    data={data.testStatusData.map(({ status, percentage }) => ({
-                      name: status,
-                      percentage,
-                    }))}
-                  />
+                  <Bar>
+                    <ProgressBar
+                      onClick={() => navigate(`system/${data.name}`)}
+                      data={data.testStatusStats.percentages.map(
+                        ({ status, percentage }) => ({
+                          name: status,
+                          percentage,
+                        }),
+                      )}
+                    />
+                  </Bar>
                 )
               },
             },
             score: {
               header: 'Score',
-              width: 155,
-              Cell: ({ index }) => {
-                //TODO@voja update this when we have some logic
-                if (index === 0) {
+              width: 200,
+              Cell: ({ data }) => {
+                if (data.score === 'good') {
                   return (
-                    <Button
-                      onClick={() => navigate('/repository-details')}
-                      variant="rounded"
-                      size="small"
-                      color="success"
-                    >
-                      Good
-                    </Button>
+                    <Bar>
+                      <Button
+                        onClick={() => navigate(`system/${data.name}`)}
+                        variant="rounded"
+                        size="small"
+                        color="success"
+                      >
+                        Good
+                      </Button>
+                    </Bar>
                   )
                 }
-                if (index === 1) {
+                if (data.score === 'bad') {
                   return (
-                    <Button
-                      onClick={() => navigate('/repository-details')}
-                      variant="rounded"
-                      size="small"
-                      color="error"
-                    >
-                      Bad
-                    </Button>
+                    <Bar>
```

Comment:
We could have this Good, bad, etc as a translation and then a method `getScoreButton`
Since the button is the same always only thing that is changed is text and color, we should handle that something like
we should also create a method in `frontend/packages/core/src/utilities/ui.ts` that will handle the score and return the color for this button
and text we can handle `{t('global.button.${score}')`

Reply:
Done